### ISSUE TITLE
[Date Importer] Can't log in to Epic

### ISSUE THREAD

**Issue by FolkSong** [darklinkpower/PlayniteExtensionsCollection#151]
### Extension name

Date Importer

### Bug Description

Whenever the plugin asks me to log in to my Epic account, I enter my username and password and then it gives me an hCaptcha test where I have to identify images containing bicycles, trains, etc. After doing this, the web page shows a red banner saying "Incorrect response, please refresh the page". I've tried many times.

At first I thought I was failing the hCaptcha, but I tried intentionally selecting the wrong pictures and gives a different error, an orange banner saying "Challenge error".

I tried logging in through an incognito window in my regular web browser and it works, but it gives me a 2-factor request instead of hCaptcha (they email me a code which I have to enter).

Steam, GOG and Origin work fine.

### Extensions log

https://gist.github.com/FolkSong/420cef81510dcb85f46a05572371c463

**Comment by darklinkpower** [comment]
I'll take a look when I can. For the moment, I think the issue should be gone if you login via the library plugin settings page

**Comment by FolkSong** [comment]
Thanks. I actually was logged in to every service to begin with, but Steam and Epic still popped up the login window. So that's maybe the bigger issue. For Steam the login worked right away though, and GOG and Origin didn't need any interaction.

I just tried it again, and now Steam works with no login windows, but Epic was the same. Finally I tried re-authenticating the main Epic library plugin. It asked me for the 2 factor code rather than the hCaptcha, and it worked. After that the Date Importer worked as well, no extra login needed. So it's an easy workaround at least.
